src/main.rs

```diff
@@ -494,63 +732,121 @@ fn get_setup(args: &[String]) -> Setup {
             }),
         };
 
+        #[cfg(not(feature = "alsa-backend"))]
+        let mixer_device = MixerConfig::default().device;
+
+        #[cfg(feature = "alsa-backend")]
         let index = match matches.opt_str("mixer-index") {
             Some(index) => {
-                warn!("--mixer-index is deprecated and will be removed in a future release.");
-                warn!("Please use --alsa-mixer-index instead.");
-                index
-                    .parse::<u32>()
-                    .expect("Mixer index is not a valid number")
+                warn!("`--mixer-index` is deprecated and will be removed in a future release.");
```

Comment:
I'm thinking of this now, but we deprecated these in 0.3.0 did we not? If so I'd be OK with removing them now, since the next release will be 0.4.0 due to API breakage.

Reply:
Can do.

Reply:
Done.